spectator/writer/udp_writer_test.go

```diff
@@ -103,7 +103,7 @@ func TestUdpWriter_Write(t *testing.T) {
 }
 
 func TestConcurrentWrites(t *testing.T) {
-	messagesPerThread := 2500
+	messagesPerThread := 1000
```

Comment:
We should maybe leave a comment here about what we think the actual top-end looks like, so we know what it is, even though we downshift to improve CI testing reliability.

Reply:
I change it down from 2500 because it was sometimes failing (though rarely).

My understanding is that it really depends on how fast the reader can consume messages on the other end so I'm not sure a test will give us an accurate figure. For example, I tried just counting the messages instead of accumulating them in an array and that changed the max threshold.

If this is a concern we should introduce a buffer.

Reply:
Interesting observation. I can see how that makes sense. Not too worried about introducing a buffer just yet - we are looking for a basic t-shirt sizing style of the performance for now.